nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/GetHDFS.java

```diff
@@ -388,18 +388,18 @@ protected void processBatchOfFiles(final List<Path> files, final ProcessContext
                 flowFile = session.putAttribute(flowFile, CoreAttributes.FILENAME.key(), outputFilename);
 
                 if (!keepSourceFiles && !getUserGroupInformation().doAs((PrivilegedExceptionAction<Boolean>) () -> hdfs.delete(file, false))) {
-                    getLogger().warn("Could not remove {} from HDFS. Not ingesting this file ...",
-                            new Object[]{file});
+                    getLogger().warn("Could not remove {} from HDFS. Not ingesting this file ...", file);
                     session.remove(flowFile);
                     continue;
                 }
 
                 session.getProvenanceReporter().receive(flowFile, file.toString());
                 session.transfer(flowFile, REL_SUCCESS);
-                getLogger().info("retrieved {} from HDFS {} in {} milliseconds at a rate of {}",
-                        new Object[]{flowFile, file, millis, dataRate});
+                getLogger().info("retrieved {} from HDFS {} in {} milliseconds at a rate of {}", flowFile, file, millis, dataRate);
+            } catch (final IOException e) {
```

Comment:
(GitHub doesn't allow me to suggest efficiently)

This way we don't handle non-GSS IOExceptions properly.

Instead of 
```java
            } catch (final IOException e) {
                handleAuthErrors(e, session, context);
            } catch (final Throwable t) {
                getLogger().error("Error retrieving file {} from HDFS due to {}", file, t);
                session.rollback();
                context.yield();
```
we should have
```java
            } catch (final Throwable t) {
                if (!handleAuthErrors(t, session, context)) {
                    getLogger().error("Error retrieving file {} from HDFS due to {}", file, t);
                    session.rollback();
                    context.yield();
                }
```